apps/meteor/client/views/room/composer/messageBox/MessageBoxActionsToolbar/MessageBoxActionsToolbar.tsx

```diff
@@ -28,6 +28,7 @@ type MessageBoxActionsToolbarProps = {
 	isRecording: boolean;
 	rid: IRoom['_id'];
 	tmid?: IMessage['_id'];
+	handleFiles?: any;
```

Comment:
Please make the type correct, restricting to the right function signature

Reply:
done sir